docs/api/auto-updater.md

```diff
@@ -50,6 +50,11 @@ The server-side setup is also different from OS X. You can read the documents of
 There is no built-in support for auto-updater on Linux, so it is recommended to
 use the distribution's package manager to update your app.
 
+## Usage
+```javascript
+var autoUpdater = require('electron').remote.autoUpdater;
```

Comment:
This code implies usage from the renderer process.

Most examples use the module in their intended process and for `autoUpdater` it is the main process.

So then it would be:

```js
const {autoUpdater} = require('electron');
```